flink-formats/flink-csv/src/main/java/org/apache/flink/formats/csv/CsvFormatOptions.java

```diff
@@ -87,11 +87,12 @@ public class CsvFormatOptions {
                             "Optional null literal string that is interpreted as a\n"
                                     + "null value (disabled by default)");
 
-    public static final ConfigOption<Boolean> WRITE_BIGDECIMAL_AS_PLAIN =
-            ConfigOptions.key("write-bigdecimal-as-plain")
+    public static final ConfigOption<Boolean> DISABLE_WRITE_BIGDECIMAL_AS_SCIENTIFIC_NOTATION =
```

Comment:
Why not just `WRITE_BIGDECIMAL_IN_SCIENTIFIC_NOTATION` without DISABLE? The boolean already incorporates this notion. (also mind IN instead of AS)